.nsprc

```diff
@@ -1,5 +1,7 @@
 {
   "exceptions": [
+    "https://nodesecurity.io/advisories/55",
+
```

Comment:
Will nsp blow its lid if a JS comment is added to a JSON file? Knowing what each of the entries are for seems important.